versions/3.0.4.md

```diff
@@ -280,7 +281,7 @@ The metadata MAY be used by the clients if needed, and MAY be presented in editi
 | <a name="info-terms-of-service"></a>termsOfService | `string` | A URL for the Terms of Service for the API. This MUST be in the form of a URL. |
 | <a name="info-contact"></a>contact | [Contact Object](#contact-object) | The contact information for the exposed API. |
 | <a name="info-license"></a>license | [License Object](#license-object) | The license information for the exposed API. |
-| <a name="info-version"></a>version | `string` | **REQUIRED**. The version of the OpenAPI document (which is distinct from the [OpenAPI Specification version](#oas-version) or the version of the API being described). |
+| <a name="info-version"></a>version | `string` | **REQUIRED**. The version of the OpenAPI Document (which is distinct from the [OpenAPI Specification version](#oas-version) or the version of the API being described or the version of the OpenAPI Description). |
```

Comment:
Do we want to say anything anywhere about "the version of the OpenAPI Description"?  AFAIK, no such thing exists, but this wording suggests it exists somewhere?  Either way, I agree that this is correct.

Reply:
In the TDC we concluded that there is no clear way to define the version of the OpenAPI Description. Maybe this is something we try to tackle in a future version.

Reply:
Can we add a parenthetical aside or a note under the table or _something_ indicating that there are no fields for the API or the overall OAD?  People tend to think they're missing something, but they're not.